tenant_schemas/models.py

```diff
@@ -49,7 +51,16 @@ def save(self, verbosity=1, *args, **kwargs):
 
         if is_new and self.auto_create_schema:
             try:
-                self.create_schema(check_if_exists=True, verbosity=verbosity)
+                created = self.create_schema(check_if_exists=True, verbosity=verbosity)
```

Comment:
Is a file missing here? It seems like you changed something inside ``create_schema`` but I don't see it in the changelist.